tests/e2e/manifest/reset.sh

```
@@ -7,6 +7,17 @@
 # Apply ElastiService
 kubectl apply -f  "$1/target-elastiservice.yaml" -n target
 
+# Wait for ElastiService to be old enough
+CRD_CONTENT=$(kubectl get elastiservices/target-elastiservice -n target -o json)
+CRD_COOLDOWN_PERIOD=$(echo ${CRD_CONTENT} | jq -r .spec.cooldownPeriod)
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)